src/univers/version_range.py

```diff
@@ -164,6 +174,33 @@ def from_versions(cls, sequence):
             constraints.append(constraint)
         return cls(constraints=constraints)
 
+    def invert(self):
+        """
+        Return the inverse of this VersionRange. For example, if this range is
+        ">=1.0.0", the inverse is "<1.0.0".
+        >>> VersionRange.from_string("vers:npm/>=1.0.0").invert()
+        NpmVersionRange(constraints=(VersionConstraint(comparator='<', version=SemverVersion(string='1.0.0')),))
+        """
+        inverted_constraints = []
+
+        if len(self.constraints) == 1 and self.constraints[0].comparator == "*":
+            # The inverse of "*" is an empty range.
+            return None
+
+        for constraint in self.constraints:
+            if constraint.comparator in INVERTED_COMPARATORS:
```

Comment:
By construction it is impossible to get an unknown comparator so late n the process, so you can safely do this instead and do not need to ever raise anything
```suggestion
            inverted_comparator = INVERTED_COMPARATORS[constraint.comparator]
```